packages/integrations/node/src/standalone.ts

```diff
@@ -39,7 +39,8 @@ export default function startServer(app: NodeApp, options: Options) {
 	const { client } = resolvePaths(options);
 	const handler = middleware(app);
 
-	const host = getResolvedHostForHttpServer(options.host);
+	// Allow to provide host value at runtime
+	const host = getResolvedHostForHttpServer(process.env.HOST !== undefined ? process.env.HOST : options.host);
```

Comment:
I think an empty string for HOST would still return true, would it make sense to update it to process.env.HOST ? process.env.HOST : options.host, instead?

Reply:
I took some time to think about the logic but we have also the `false` case which is falsy and would skip the runtime variable

Reply:
I can manage the condition with better edge cases management:
- `undefined` or empty string would mean no runtime variable
- else we have a value (`true`, `false`, `string`)

Reply:
What do you think ?

Reply:
> I can manage the condition with better edge cases management:
> 
> * `undefined` or empty string would mean no runtime variable
> * else we have a value (`true`, `false`, `string`)

@Scttpr 
That looks ideal!

Reply:
> > I can manage the condition with better edge cases management:
> > 
> > * `undefined` or empty string would mean no runtime variable
> > * else we have a value (`true`, `false`, `string`)
> 
> @Scttpr That looks ideal!

Fixed, don't hesitate if it is not fine :)

Reply:
@Scttpr Looking all set!